services/cards-publication/src/main/java/org/opfab/cards/publication/services/CardProcessingService.java

```diff
@@ -129,7 +129,9 @@ public void patchCard(String id, Card cardPatch, Optional<CurrentUserWithPerimet
     }
 
     public void processUserCard(Card card, CurrentUserWithPerimeters user, Optional<Jwt> jwt) {
-        card.setPublisherType(PublisherTypeEnum.ENTITY);
+        if (card.getPublisherType() != PublisherTypeEnum.USER) {
+            card.setPublisherType(PublisherTypeEnum.ENTITY);
+        }
         this.cardValidationService.validate(card);
         if (!authorizeToSendCardWithInvalidProcessState)
```

Comment:
Add a unit test in CardProcessingServiceShould 

Example : send a card with Publisher typ USER and user login and check it is ok

Reply:
Done.